py/__init__.py

```diff
@@ -93,6 +93,10 @@ class PathHolders:
     StringCodec = ImportPathHolder("StringCodec", "protocol.builtin", )
     ListLongCodec = ImportPathHolder("ListLongCodec", "protocol.builtin")
     ListIntegerCodec = ImportPathHolder("ListIntegerCodec", "protocol.builtin")
+    ListCNBooleanCodec = ImportPathHolder("ListCNBooleanCodec", "protocol.builtin")
```

Comment:
does python implement sql yet? what is this change for?

Reply:
I removed changes from the Python and NodeJS parts. 

However,  perhaps I missed a couple of other things. Should I need the new CN codecs to `util.py`, `protocol-schema.json`, and `custom-codec-schema.json`?

Reply:
Well, this is not needed until they are being used in the protocol definitions as parameter types but to be future-proof, we might add these new types to `binary/__init__.py`, `binary/__util__.py` and schema files. I believe each language could add it to its `lang/__init__.py` file because each one uses a different style for this file for their own needs. Maybe we can just add them to `java/__init__.py`

Leaving it as it is is also fine for me

Reply:
@mdumandag I think, we can expose these classes a little bit later, once I finish with the rest SQL types (double, time, etc.), which would require more custom codecs.